cypress_test/integration_tests/desktop/calc/top_toolbar_spec.js

```diff
@@ -16,6 +16,49 @@ describe('Top toolbar tests.', function() {
 		helper.afterAll(testFileName);
 	});
 	
+	it('Clear Direct formatting.', function() {
+		cy.get('#tb_editbar_item_bold')
+			.click();
+
+		calcHelper.selectAllMobile();
+
+		cy.get('#copy-paste-container table td b')
+			.should('exist');
+
+		cy.get('#tb_editbar_item_reset')
+			.click();
+
+		calcHelper.selectAllMobile();
+
+		cy.get('#copy-paste-container table td b')
+			.should('not.exist');
+		
+	});
+
+	it('Apply font style.', function() {
+		cy.get('#tb_editbar_item_fonts')
+			.click();
+
+		cy.contains('.select2-results__option','Likhan')
+			 .click({force: true});
```

Comment:
I think the test fails because this 'Likhan' font is not there on the building machine. You should not use some exotic fonts, which might be only on your computer and not on others' computers. Try to use a more common font name here.